### PR TITLE
fixes the expiry_date exclusion

### PR DESCRIPTION
**Pull request by bfloreaDVSA**

### CODE REVIEW

**Review by dpanzaru-deloitte — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -91,18 +90,6 @@
             <artifactId>aws-java-sdk-dynamodb</artifactId>
             <version>1.11.718</version>
         </dependency>
-        <dependency>
```
Comment (dpanzaru-deloitte) [inline pom.xml:94]: Why was this deleted? I believe only the duplicate should be removed.

Comment (bfloreaDVSA) [reply pom.xml:94]: it is duplicated, check the first dependency :)

**Review by jgimber-deloitte — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (dpanzaru-deloitte) [reply pom.xml:94]: There it has scope test, not compile
